apps/judicial-system/backend/src/app/modules/case/models/case.model.ts

```diff
@@ -64,122 +89,189 @@ export class Case extends Model<Case> {
+   **********/
   @Column({
     type: DataType.STRING,
     allowNull: false,
   })
   @ApiProperty()
   accusedNationalId: string
 
+  /**********
+   * The name of the accused
+   **********/
   @Column({
     type: DataType.STRING,
     allowNull: true,
   })
   @ApiProperty()
   accusedName: string
 
+  /**********
+   * The address of the accused
+   **********/
   @Column({
     type: DataType.STRING,
     allowNull: true,
   })
   @ApiProperty()
   accusedAddress: string
 
+  /**********
+   * The gender of the accused
+   **********/
   @Column({
     type: DataType.ENUM,
     allowNull: true,
     values: Object.values(CaseGender),
   })
   accusedGender: CaseGender
 
+  /**********
+   * The name of the accused´s defender - optional
+   **********/
   @Column({
     type: DataType.STRING,
     allowNull: true,
   })
   @ApiProperty()
   defenderName: string
 
+  /**********
+   * The email address of the accused´s defender - optional
+   **********/
   @Column({
     type: DataType.STRING,
     allowNull: true,
   })
   @ApiProperty()
   defenderEmail: string
 
+  /**********
+   * The phone number of the accused's defender - optional
+   **********/
   @Column({
     type: DataType.STRING,
     allowNull: true,
   })
   @ApiProperty()
   defenderPhoneNumber: string
 
+  /**********
+   * Indicates whether the prosecutor's request should be sent to the accused's defender
+   * when a court date has been assigned to the case - optional
+   **********/
   @Column({
     type: DataType.BOOLEAN,
     allowNull: true,
   })
   @ApiProperty()
   sendRequestToDefender: boolean
 
+  /**********
+   * The surrogate key of the court assigned to the case
+   **********/
   @Column({
     type: DataType.STRING,
     allowNull: true,
   })
   @ApiProperty()
   courtId: string
 
+  /**********
+   * The court assigned to the case
+   **********/
   @BelongsTo(() => Institution, 'courtId')
   @ApiProperty({ type: Institution })
   court: Institution
 
+  /**********
+   * The lead investigator assigned to the case - usually name and title -
+   * optional for most case types
+   **********/
   @Column({
     type: DataType.STRING,
     allowNull: true,
   })
   @ApiProperty()
   leadInvestigator: string
 
+  /**********
+   * The date and time the accused was arrested - optional
+   **********/
   @Column({
     type: DataType.DATE,
     allowNull: true,
   })
   @ApiProperty()
   arrestDate: Date
 
+  /**********
+   * The prosecutor's requested court date and time
+   **********/
   @Column({
     type: DataType.DATE,
     allowNull: true,
   })
   @ApiProperty()
   requestedCourtDate: Date
 
+  /**********
+   * The prosecutor's requested ruling expiration date and time - example: the end of custody
+   * in custody cases - optional for some case types
+   **********/
   @Column({
     type: DataType.DATE,
     allowNull: true,
   })
   @ApiProperty()
-  requestedCustodyEndDate: Date
+  requestedValidToDate: Date
 
+  /**********
```

Comment:
I think we should either finish filling these in or remove the empty comments

Reply:
I will fill in the comments as I think they are very important and increasingly so.